### PR TITLE
Item rebase

### PR DESCRIPTION
**Pull request by Ibrahimhizeoui**

### CODE REVIEW

File: tests/.cache/billogram.com_162a626ed36b7cc87e1efcfc4e27f9a630b53c2b [hunk 1]
```diff
@@ -0,0 +1 @@
+s:25:"{"status":"OK","data":{}}";
```
Comment (Nyholm) [inline tests/.cache/billogram.com_162a626ed36b7cc87e1efcfc4e27f9a630b53c2b]: This is never used

File: tests/.cache/billogram.com_162a626ed36b7cc87e1efcfc4e27f9a630b53c2bheaders.txt [hunk 2]
```diff
@@ -0,0 +1 @@
...Content-Type":["application\/json"],"Transfer-Encoding":["chunked"],"Connection":["keep-alive"],"Server":["billogram.com"],"Last-Modified":["Tue, 01 Aug 2017 15:29:04 GMT"],"Expires":["Tue, 01 Aug 2017 15:29:04 GMT"],"Vary":["Cookie"],"Cache-Control":["max-age=0"],"Set-Cookie":["dtCookie=8$294C144A344550682BEDC41B71F229B8|RUM+Default+Application|1; Path=\/; Domain=.billogram.com; secure"],"Content-Security-Policy":["default-src https: data: wss: 'unsafe-inline' 'unsafe-eval'"],"X-Frame-Options":["SAMEORIGIN"],"X-Xss-Protection":["1; mode=block"],"X-Content-Type-Options":["nosniff"],"Strict-Transport-Security":["max-age=31536000; includeSubdomains; preload"],"Public-Key-Pins":["pin-sha256=\"+qVQvnt5yb253ksCTDShLi4YF0vAPSNklg\/lZZBVP\/8=\"; pin-sha256=\"wdVM\/85nxkG2Y1\/J\/qYfsIuCEsj8jzERJ3Aw7yAeDSw=\"; pin-sha256=\"KXTSUWRog1JWXRETuXpaNxwBb70GhK2DPbxT48pqGM0=\"; pin-sha256=\"cbRybe3tXRUTuqwtSimQpF9niYVqDxVCgzCBOyGLJgo=\"; pin-sha256=\"lPY3Z7kFEInh8+8Qgsinuj2eME1bQOVNnVc13s3TYI0=\"; pin-sha256=\"b8WqC0bsigc3akPYzfSUBKnI0qRfz3G4EWFwhS8305k=\"; pin-sha256=\"UPYaBL6pD2gw8CIQPLZh16\/DG+16iIrph3ZUy6fKx7w=\"; pin-sha256=\"IOBCdzKsvdCTAb\/L7+3VIMtG\/doHybMnJL4HFShbzsM=\"; max-age=2592000"]}
```
Comment (Nyholm) [inline tests/.cache/billogram.com_162a626ed36b7cc87e1efcfc4e27f9a630b53c2bheaders.txt]: This file is never used

File: tests/.cache/billogram.com_162a626ed36b7cc87e1efcfc4e27f9a630b53c2b (hunk 1 above)
Comment (Ibrahimhizeoui) [inline tests/.cache/billogram.com_162a626ed36b7cc87e1efcfc4e27f9a630b53c2b]: done

File: tests/.cache/billogram.com_162a626ed36b7cc87e1efcfc4e27f9a630b53c2bheaders.txt (hunk 2 above)
Comment (Ibrahimhizeoui) [inline tests/.cache/billogram.com_162a626ed36b7cc87e1efcfc4e27f9a630b53c2bheaders.txt]: done

**Pull request merged**